include/allegro5/platform/allegro_internal_sdl.h

```diff
@@ -23,7 +23,6 @@ typedef struct ALLEGRO_DISPLAY_SDL
    SDL_GLContext context;
 } ALLEGRO_DISPLAY_SDL;
 
-typedef struct ALLEGRO_SYSTEM_INTERFACE ALLEGRO_SYSTEM_INTERFACE;
```

Comment:
Why exactly did you remove this?

Reply:
This struct is already typedefed in `internal/aintern_system.h`; it's not really related to `allegro_internal_sdl.h` and it was breaking the build on some old GCC in one of my toolchains. It seemed to me that it was here by mistake and while newer compilers seem to be smart enough to ignore duplicated typedef, I thought that it's still cleaner to just take this out.

Reply:
Ok, thanks for clarifying that for me. I agree it's probably better to remove that duplicate typedef.